controls/V-75569.rb

```diff
@@ -43,5 +55,16 @@
 \"/home/smithj\".
 
 # chmod 0740 /home/smithj/.<INIT_FILE>"
+
+  IGNORE_SHELLS = NON_INTERACTIVE_SHELLS.join('|')
+
+  findings = Set[]
```

Comment:
Please use the `more_permissive_than` matcher I created rather than this older method

Reply:
You will find it in the `File` resource

Reply:
```
be_more_permissive_than(mode)
be_more_permissive_than(mode) takes the maximum desired mode - in octal format (‘0644’ or ‘0777’) - of your file
as a String and returns a Boolean. 
It will return true if your file has a mode with greater permissions than specified.

describe file('/etc/passwd') do
  it { should_not be_more_permissive_than('0644') }
  it { should be_more_permissive_than('0000') }
end```

Reply:
using the new matcher, for some reason it looks like this is the appropriate code `next if !file(dot_file).more_permissive_than?('0740')` instead of `next if file(dot_file).more_permissive_than?('0740')` (Note the !) to find files that are more permissive